dotnet/src/IntegrationTests/IntegrationTests.csproj

```diff
@@ -35,6 +35,7 @@
     <None Remove="skills\FunSkill\Joke\skprompt.txt" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Azure.AI.OpenAI" VersionOverride="2.0-beta.2" />
```

Comment:
Need to resolve version in one direction or the other.  Since Agent integration test lives here, pinning to old version is a non-starter (same for  your second connector).

Reply:
This is not a real scenario, in this project you don't need a package reference you just use the Project relative path.